plugins/org.eclipse.n4js.cli/pom.xml

```diff
@@ -0,0 +1,211 @@
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>com.googlecode.addjars-maven-plugin</groupId>
+				<artifactId>addjars-maven-plugin</artifactId>
+				<version>${addjars-maven-plugin.version}</version>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>add-jars</goal>
+						</goals>
+						<configuration>
+							<resources>
+								<resource>
+									<directory>${project.build.directory}/lib</directory>
+								</resource>
+							</resources>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<version>${maven-shade-plugin.version}</version>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<transformers>
+								<transformer
+									implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+									<mainClass>org.eclipse.n4js.cli.RunServer</mainClass>
+								</transformer>
+								<transformer
+									implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+									<resource>plugin.properties</resource>
+								</transformer>
+							</transformers>
+							<filters>
+								<filter>
+									<artifact>*:*</artifact>
+									<excludes>
+										<exclude>META-INF/INDEX.LIST</exclude>
+										<exclude>META-INF/*.SF</exclude>
+										<exclude>META-INF/*.DSA</exclude>
+										<exclude>META-INF/*.RSA</exclude>
+										<exclude>.options</exclude>
+										<exclude>.api_description</exclude>
+										<exclude>*.profile</exclude>
+										<exclude>*.html</exclude>
+										<exclude>about.*</exclude>
+										<exclude>about_files/*</exclude>
+										<exclude>plugin.xml</exclude>
+										<exclude>modeling32.png</exclude>
+										<exclude>systembundle.properties</exclude>
+										<exclude>profile.list</exclude>
+										<exclude>**/*._trace</exclude>
+										<exclude>**/*.g</exclude>
+										<exclude>**/*.mwe2</exclude>
+										<exclude>**/*.xtext</exclude>
+									</excludes>
+								</filter>
+								<filter>
+									<artifact>org.eclipse.n4js:*log4j*</artifact>
+									<includes>
+										<include>**</include>
+									</includes>
+								</filter> 
+								<filter>
+									<artifact>org.eclipse.n4js:*com.google.inject*</artifact>
+									<includes>
+										<include>**</include>
+									</includes>
+								</filter> 
+							</filters>
+							<shadedArtifactAttached>true</shadedArtifactAttached>
+							<shadedClassifierName>${cli.fat.jar.classifier}</shadedClassifierName>
+							<minimizeJar>true</minimizeJar>    
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			
+			
+			<!-- copy the longname-jar to a usable shorthand name. -->
```

Comment:
Such a one-line comment would be helpful above each `<plugin>` tag, to explain what each plugin invocation is doing.